spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#show', type: :feature do
```

Comment:
- [ ] Hello, kindly make sure your test case only include those include in the project requirement, check below for guidance

![image](https://github.com/BilalLiaquat7/blogapp/assets/112261237/bb727f40-3044-440d-a4b2-9391b6470fa7)